core/src/lib.rs

```diff
@@ -131,67 +131,67 @@ unsafe impl<T> Field for Identity<T> {
     }
 }
 
-/// Projects a type to the given field
+/// Projects a type to a given `Field`
 pub trait ProjectTo<F: Field> {
-    /// The projection of the type, can be used to directly access the field
+    /// Direct access to the field
     type Projection;
 
-    /// projects to the given field
+    /// Project to a given `Field`
     fn project_to(self, field: F) -> Self::Projection;
 }
 
-// TODO: reword this documentation, tis bad
-/// Projects a type to the given field
+/// Project a given Field onto a procedurally generated Field using an `unsafe`
```

Comment:
Would it be better to quote Field?  That is, \`Field\` instead of plain 'Field'?